### DISCUSSION TITLE
advice for running docker image within cronjob

### DISCUSSION THREAD

**Discussion by aledj2** [icloud-photos-downloader/icloud_photos_downloader#475] [Q&A] [answered]
I'm able to run the docker image (thanks v.much!) but I'd like to be able to run this as a cronjob.
I think the issue is it does not use the existing credentials, and asks for 2FA again. 

As the existing docker commands are run as sudo I'd have thought should have permissions to read existing cookies?

**Comment by marcomusso** [comment, marked as answer]
I just did it and if you mount the directory where it stores the cookies then 2FA is not asked anymore (see the docker command example in the README).

**Comment by AndreyNikiforov** [comment]
You can also use `--watch-with-interval` param to run your docker as a services and even use [webui](https://icloud-photos-downloader.github.io/icloud_photos_downloader/webui.html) for pass/mfa
